src/components/CheckListItem.jsx

```diff
@@ -1,11 +1,11 @@
 import React from "react";  
 import './CheckListItem.css'
 const CheckListItem = (props) => {
-        const {heading, description} = props.data
+        const {id, text, completed} = props.data
     return (
         <>
-        <h3 style={{color: 'green'}} className='checkbox'> {heading}</h3> <p>{description}</p>
-        <input type='checkbox'/>
+        <h3 style={{color: 'green'}} className='checkbox'> {id}. {text}  <input type='checkbox' checked = {completed} /></h3>
```

Comment:
Nice I like how you passed in the checked attribute and then use that status to check if it is completed or not? what are you using id here for? is it literally just to display id next to the text?